### DISCUSSION TITLE
Windows 10 VS 11

### DISCUSSION THREAD

**Discussion by happy-spark** [community/community#34069] [Other Feature Feedback, Questions, & Ideas]
Windows 10 VS 11 

Which do you think is better?

**Comment by RaheemAmer** [comment]
"In short, updating to Windows 11 means a more snappy experience where apps load faster and your PC wakes from sleep with ease. Windows 10 fast, but Windows 11 is just a bit faster"

Check this comparison for more detailed info
- https://www.digitaltrends.com/computing/windows-11-vs-windows-10/

**Reply by happy-spark** [reply]
Thank you!

**Comment by HannaPleshko** [comment]
11

**Comment by heipiao233** [comment]
3. Windows 1.0

**Comment by Habeel06** [comment]
both are good!

**Comment by LegationPro** [comment]
windows 11 is definitely nice
both are quite good must say i don't regret installing it and i use it everyday now
